plugins/warpsync/plugin.go

```diff
@@ -20,7 +20,7 @@ import (
 var (
 	PLUGIN   = node.NewPlugin("WarpSync", node.Enabled, configure)
 	log      *logger.Logger
-	warpSync = warpsync.New(300)
+	warpSync = warpsync.New(25)
```

Comment:
Maybe we should add a setting in the config? What do you think?

Reply:
lets leave it like this for now, otherwise people are going to mess with it too much

Reply:
:+1: